modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/raylife-ap-react-fragments/fragments/applications-status/index.js

```diff
@@ -61,36 +61,55 @@ export default function () {
 				reviewed: '#4C84FF',
 				underwriting: '#B5CDFE',
 			};
+
 			const cols = [
-				[CONSTANTS.STATUS.OPEN, getTotalCount(openApplicationsResults)],
+				[
+					CONSTANTS.STATUS.OPEN,
+					getTotalCount(openApplicationsResults),
+					CONSTANTS.STATUS.OPEN.charAt(0).toUpperCase() +
```

Comment:
Could you move this code to a generic function? Please do this for all statuses. I think it would be better to move it to utils.